modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -47,19 +47,20 @@ def submit_supporting_documents
       def get_attachment_ids_and_form(parsed_form_data)
         form_id = get_form_id
         form = "IvcChampva::#{form_id.titleize.gsub(' ', '')}".constantize.new(parsed_form_data)
-        attachment_ids = [form_id]
+        attachment_ids = []
 
-        if form_id == 'vha_10_10d'
-          parsed_form_data['applicants'].each do |applicant|
-            next unless applicant.key?('applicant_supporting_documents')
+        allowed_form_ids = %w[vha_10_10d vha_10_7959c]
```

Comment:
Is there much benefit to whitelisting specific form IDs? 

Seems like the presence of the `supporting_docs` key in the form data carries the same information (because we have to intentionally add that on the frontend). 

If it's not serving much purpose we could remove it to make this one less place we have to regularly update? Just a thought though

Reply:
Ya, I removed it